### PR TITLE
Packet in data

### PR DESCRIPTION
**Pull request by oqu**
Do you need more unit-test ?
I only updated the one you already written for ParsePacketIn

### CODE REVIEW

**Comment by Kmotiko** [comment]
> Do you need more unit-test ?
> I only updated the one you already written for ParsePacketIn

No, you don't have to do it.
You exactly did what I was hoping you would do.
Thank you so much. 

Now, I still haven't been able to confirm unit-test code which you updated completely.
I'll merge your pull-request after do it.

File: ofprotocol/ofp13/ofp13_parser_test.go [hunk 1]
```diff
@@ -3972,7 +3973,7 @@ func TestSerializeMeterBandDscpExperimenter(t *testing.T) {
 /* OfpPacketIn                                       */
 /*****************************************************/
 func TestParsePacketIn(t *testing.T) {
-	packet := []byte{
+	packetWithoutPacketIn := []byte{
```
Comment (Kmotiko) [inline ofprotocol/ofp13/ofp13_parser_test.go]: I think this byte array represents "PacketIn" packet data, so  "packetWithoutPacketIn" is not suitable for this variable
I think that other names are preferable, for example "packetInWithoutReceivedData".
What do you think about ?

File: ofprotocol/ofp13/ofp13_parser_test.go [hunk 2]
```diff
@@ -3990,19 +3991,23 @@ func TestParsePacketIn(t *testing.T) {
 		0xff, 0xff, 0xff, 0xfe, // Value
 		0x00, 0x00, 0x00, 0x00, // Padding
 		0x00, 0x00, // Padding
-		0xff, 0xff, 0xff, 0xff, 0xff, 0xff, //
-		0x00, 0x23, 0x20, 0x90, 0x00, 0x00, //
-		0x80, 0x35,
+	}
+
+	packetIn := []byte{
```
Comment (Kmotiko) [inline ofprotocol/ofp13/ofp13_parser_test.go]: Just like line 3976, I prefer another variable name for this byte array.

**Comment by oqu** [comment]
Sure. I will update and push.

**Comment by oqu** [comment]
Done

**Comment by Kmotiko** [comment]
Thank you for the update.
I'll confirm it :)

**Comment by Kmotiko** [comment]
I merge your Pull Request.
Thank you @oqu !!

**Pull request merged**